docs/welcome.md

```diff
@@ -4,35 +4,57 @@ title: Welcome
 slug: /welcome
 ---
 
-# Amplication Documentation
+# Getting Started with Amplication
 
 ---
 
-Hi, welcome to Amplication's documentation.
+Hello and welcome to Amplication's developer documentation.
 
-[Amplication](/about/) is an open‑source development tool. It helps professional Node.js developers to develop quality Node.js applications without spending time on repetitive coding tasks.
+[Amplication](/about/) is an open-source backend development tool. It accelerates the development of Node.js applications by auto-generating fully-functional apps.
 
-Amplication auto-generates fully functional apps based on TypeScript and Node.js.
+Build critical apps faster. Streamline your development process and increase productivity so you can focus on what matters most: **creating awesome applications**.
 
-## What Amplication helps you generate
+## Key Features of Amplication
 
-Generated apps include NestJS, Prisma, [REST API](/api/#rest-api), [GraphQL API](/api/#graphql-api), a React admin UI, authentication, authorization, and logging.
+Amplication helps you build a fully functional backend in minutes. Just define your [data models](/how-to/create-entity/), roles, [permissions](/how-to/set-access-permissions/), and Amplication will generate a Node.js application with everything you need.
```

Comment:
The punctuation is a bit off IMO. Needs another comma and then perhaps a dash:
" Just define your data models, roles, and permissions - and Amplication..."